Content.Client/GameObjects/EntitySystems/ExamineSystem.cs

```diff
@@ -141,6 +173,28 @@ public async void DoExamine(IEntity entity)
                     break;
                 }
             }
+
+            var size = Vector2.ComponentMax((minWidth, 0), panel.CombinedMinimumSize);
```

Comment:
This is the initial size of the popup control and I've moved it down after we have gotten our message